compiler/onecc-docker/onecc-docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+if ! command -v docker > /dev/null
+        then
+        echo "docker must be installed"
+        exit 0
+fi
+
+if ! command -v python3 > /dev/null
+        then
+        echo "python3 must be installed"
+        exit 0
+fi
+
+if command -v onecc > /dev/null 
+        then
+        version=( $(onecc --version) )
+        exec "python3" "onecc-docker.py" "${version[2]}" "$@" 
```

Comment:
(Q) Isn't it possible to receive the `onecc` version in `onecc-docker.py` file?

Reply:
I thought that onecc --version command should be checked when onecc is installed in ubuntu system
Do you mean that just excute python3 onecc-docker.py $@ command and check installation of onecc in onecc-docker.py?

Reply:
Yes. I think that you can get the status whether the `onecc` is installed in `onecc-docker.py` file.